vars/unitTest.groovy

```diff
@@ -84,12 +87,14 @@ def call(Map config = [:]) {
     stashes.add("${target_compiler}-install")
     stashes.add("${target_compiler}-build-vars")
   }
+  
+  def valgrind = config.get('valgrind', '')
```

Comment:
We want the major defaults to come from the `stage_info` map, so that they can be set either by setting the contents fo the stage name or by environment variables in the case of a matrix job.

To be able to move to Jenkins matrix notation all steps inside the stage need to have identical parameters and have defaults set by either parsing the stage name environment variable or environment variables set for by a Jenkins matrix.